src/localization/sim_imu_driver.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+import rospy
+import numpy as np
+from mrover.msg import ImuAndMag
+from sensor_msgs.msg import Imu, MagneticField
+from geometry_msgs.msg import PoseWithCovarianceStamped, Vector3Stamped
+import message_filters
+from esw.imu_driver import get_covariances, publish_mag_pose
+
+
+class SimIMUDriver:
+    """
+    This node is for use in sim - it merges the /imu/imu_only & /imu/mag_only messages
+    published by gazebo & publish to /imu/data, to replicate the behavior of imu_driver.py
+    """
+
+    imu_pub: rospy.Publisher
+    mag_pose_pub: rospy.Publisher
+
+    orientation_covariance: np.ndarray
+    gyro_covariance: np.ndarray
+    accel_covariance: np.ndarray
+    mag_pose_covariance: np.ndarray
+
+    world_frame: str
+
+    def __init__(self):
+        (
+            self.orientation_covariance,
+            self.gyro_covariance,
+            self.accel_covariance,
+            self.mag_pose_covariance,
+        ) = get_covariances()
```

Comment:
are the parenthesis like a style thing here? I personally think they're kinda confusing because it almost seemed like they were parameters to the init function but I guess if that the style rule then its fine. Could also be the way github renderes the diff

Reply:
this is black's choice, also I personally think this is much better than a 150 char line